### ISSUE TITLE
Explore tool bug

### ISSUE THREAD

**Issue by kbailey-noaa** [asascience-open/IOOS-EDS#82]
When choosing comparison and waves:
Chose 52201 - For some reason there are 2 items in the legend for this one station but only 1 curve.
Same for water temperature - there is an extra legend label.

**Comment by kknee** [comment]
looks like there are actually two curves drawing directly on top of each other.  will investigate.

**Comment by kknee** [comment]
looks like this is an issue for all CDIP stations.

**Comment by kbailey-noaa** [comment]
I found another small issue in the Explore tool with the CDIP stations, so including it here:
In the map, I activated the NDBC & Partner stations and the Global WW3 (height) layers.  I hit the 'explore' link in the pop-up wndow and under 'available stations/models' it says 3 items selected (should only be 2 - one for the model, one for the 51210 station).  It turns out the 51210 station is selected twice,  likely because it's also a CDIP layer station.  This is a bit confusing.  I realize it's because of the overlap in the NDBC layer and the CDIP layer, and how some NDBC stations are also labeled 'CDIP', from which a new layer was created.  I think if we grab the CDIP stations directly from Scripps (which we should do anyway) it might resolve this issue.  See issue #30. 

Also I noticed when I hit that little 'explore' link and it takes me to the 'explore tool' page the Map button (upper right) is still highlighted.  Shouldn't the Tools button be highlighted?
